python/ray/train/__init__.py

```diff
@@ -13,6 +13,7 @@
     world_size,
 )
 from ray.train.trainer import Trainer, TrainingIterator
+from ray.air.config import CheckpointConfig
```

Comment:
Do we want to expose this alias? Or just have users directly use the one from `air`?

Reply:
This PR is just to match previous behavior which got removed in #27005 (the current Ray Train docs show `from ray.train import CheckpointConfig`).

We can remove the alias in a follow up.

Reply:
Okay got it should we add a deprecation warning [here](https://github.com/amogkam/ray/blob/5febe7488b8f014a5d0b5399a2b8b9f4f09e7dd3/python/ray/train/checkpoint.py) for this?

Reply:
Ah  that one is for `train.CheckpointStrategy`, not `train.CheckpointConfig`